### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
Added login page

### CODE REVIEW

File: src/components/NavBar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import React from "react";
+import FavoriteBorderOutlinedIcon from "@mui/icons-material/FavoriteBorderOutlined";
+import ShoppingCartOutlinedIcon from "@mui/icons-material/ShoppingCartOutlined";
+import SearchOutlinedIcon from "@mui/icons-material/SearchOutlined";
+import { useLocation, Link } from "react-router-dom";
+
+const Navbar = () => {
+  const location = useLocation();
+  return (
+    <>
```
Comment (lomesshh) [inline src/components/NavBar/Navbar.jsx:11]: you haven't used location on this page remove it if not required

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Navbar from "../../components/NavBar/Navbar";
+import "./Home.css";
+import men from "../../assets/images/men.webp";
+import women from "../../assets/images/women.webp";
+import kids from "../../assets/images/kids.webp";
+import summer from "../../assets/images/summer.webp";
+import winter from "../../assets/images/winter.webp";
+import arrival from "../../assets/images/arrival.webp";
+import trending from "../../assets/images/trending.webp";
```
Comment (lomesshh) [inline src/pages/Home/Home.jsx:10]: Suggestion:
host the images and then use them directly in img src="" tag

**Review by lomesshh — COMMENTED** [review]
overall code looks clean 
rest looks good to merge

**Review by lomesshh — APPROVED** [review]

File: src/components/NavBar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+.Nav-header {
+  display: flex;
+  align-items: center;
+  background-color: var(--bg-lightWhite);
+  height: 12vh;
+  padding: 2.4rem 4.8rem;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  /* box-shadow: 0 4px 12px 0 rgba(0 , 0 ,0 , .8); */
```
Comment (romabulani) [inline src/components/NavBar/Navbar.css:12]: Remove the comment if not required

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+              <img src={kids} alt="Kid's Fashion" />
+            </article>
+            <article className="category-article">
+              <div className="category-overlay">Summer Collections</div>
+              <img src={summer} alt="Summer Fashion" />
+            </article>
+            <article className="category-article">
+              <div className="category-overlay">Winter Collections</div>
+              <img src={winter} alt="Winter Fashion" />
+            </article>
```
Comment (romabulani) [inline src/pages/Home/Home.jsx:58]: You can use one div here in map function by keeping the categories data in backend

**Review by romabulani — APPROVED** [review]
-Liked the folder structure
- Have given some suggestions if you have time, good to merge

File: src/pages/Home/Home.jsx (hunk 4 above)
Comment (Aman-Jain1996) [reply src/pages/Home/Home.jsx:58]: Yes , roma will do all these in upcoming PR's

**Pull request merged**